lib/Charts/ChartRenderer.js

```diff
@@ -516,49 +679,83 @@ function findSelectedData(data, x) {
       Math.abs(current.x - x) < Math.abs(previous.x - x) ? current : previous
     )
   );
+  if (closestXPoints.length === 0) return [];
+  let selectedPoints = null;
+  let isSelectedArray = null;
+
+  function compareUsingDates(a, b) {
+    const diffTime = Math.abs(a.x.getTime() - b.getTime());
+    const diffDays = Math.ceil(diffTime / (1000 * 60 * 60 * 24));
+    return diffDays < 3;
```

Comment:
This `3`, and the `1e-8` below, are pretty arbitrary and likely to be give very bad results for some datasets. Is there any way to ask D3 for the X delta that corresponds to, say, 3 pixels? Cause I think something like that would be ideal.

Reply:
Have made a 3 pixel range